src/status_im/contexts/wallet/send/transaction_progress/view.cljs

```diff
@@ -25,6 +26,31 @@
     (some (fn [[_k v]] (= (:status v) :confirmed)) transaction-details)   :confirmed
     :else                                                                 nil))
 
+(defn show-save-address-bottom-sheet
+  ([] (show-save-address-bottom-sheet :blue))
+  ([_color]
+   (debounce/throttle-and-dispatch [:open-modal :screen/wallet.save-address] 1000)))
+
+(defn footer
+  [{:keys [color leave-page]}]
+  (let [save-address-visible? false]
+    [rn/view {:style style/footer}
+     (when save-address-visible?
+       [quo/button
+        {:type                :grey
+         :icon-left           :i/contact-book
+         :accessibility-label :save-address
+         :container-style     (style/footer-button save-address-visible?)
+         :on-press            (partial show-save-address-bottom-sheet color)}
+        (i18n/label :t/save-address)])
+     [quo/button
+      {:customization-color color
```

Comment:
perhaps changing the variable name to `customization-color` or `user-color` is more descriptive here 👍

Reply:
This `color` comes from re-frame db directly, so I think we should just use it as is:

```clj
        {:keys [color]} (rf/sub [:wallet/current-viewing-account])
```